### DISCUSSION TITLE
Mention Apple’s February 2021 authentication requirement in Continuous Integration docs

### DISCUSSION THREAD

**Discussion by sonicdoe** [fastlane/fastlane#18124] [Ideas] [answered]
[From Apple’s support article on two-factor authentication](https://developer.apple.com/support/authentication/):

> Starting February 2021, two-factor authentication or two-step verification will be required for all users to sign in to App Store Connect.

The [Continuous Integration](https://docs.fastlane.tools/best-practices/continuous-integration/#two-step-or-two-factor-auth) page only mentions the older February 2019 requirement:

> Note: [Apple announced that as of February 27th 2019](https://developer.apple.com/news/?id=02202019a), it is enforcing 2-factor authentication on developer Apple IDs with the "Account Holder" role.

I’d suggest to update this section to mention the February 2021 requirement (and drop the old note).

**Comment by max-ott** [comment, marked as answer]
Fastlane is open source. Please open a pull request and change the docs in it.

**Reply by sonicdoe** [reply]
[As I’ve said in #18123](https://github.com/fastlane/fastlane/discussions/18123#discussioncomment-512108), I hope my post didn’t come across as dismissive. As with that question, this one was also resolved by https://github.com/fastlane/docs/pull/1032 🙌
